src/fileActions.jsx

```diff
@@ -479,10 +472,10 @@ export const EditPermissionsModal = ({ selected, path }) => {
                 />}
                 <Form isHorizontal>
                     <FormSection title={_("Ownership")}>
-                        <FormGroup label={_("Owner")} fieldId="edit-permissions-owner">
+                        <FormGroup label={_("User")} fieldId="edit-permissions-user">
```

Comment:
This requires some @garrett signoff I suppose. It makes a lot more sense from an unixy admin perspective. But nautilus calls this `Owner`.

Reply:
Good call.  I used "user" for the property instead of "owner" because I felt it jived better with "uid", but that doesn't imply that we need to change the UI.